doc/transcodingoptions.md

```diff
@@ -89,21 +89,21 @@ An example of a full JSON configuration:
         "bitrate": 5000000,
         "fps": 30000,
         "fpsDen": 1001,
-        "profile": H264High,
-        "gop": "2",
+        "profile": "H264High",
```

Comment:
Maybe [constant](https://github.com/livepeer/lpms/blob/master/ffmpeg/videoprofile.go#L40) should be lowercase? or it doesn't matter?

Reply:
We lowercase it, so I guess it doesn't matter https://github.com/livepeer/lpms/blob/f4dcebdca29cd2a0609d0317b1bd06e12af904bc/ffmpeg/videoprofile.go#L195